src/components/protocol_handler/test/protocol_handler_tm_test.cc

```diff
@@ -1057,10 +1057,6 @@ TEST_F(ProtocolHandlerImplTest, MalformedLimitVerification) {
   AddConnection();
   AddSession();
 
-  // Expect malformed notification to CH
-  EXPECT_CALL(session_observer_mock, OnMalformedMessageCallback(connection_id))
-      .Times(1);
```

Comment:
@vlantonov I think this `EXPECT_CALL` is the main goal of this test. What are we expected now here?